ansible/roles/software/dashboard/dashboard-db-migrations/tasks/main.yml

```diff
@@ -2,12 +2,12 @@
 
 - name: Temporarily disabling keeping sessions in DB
   replace:
-    dest: "{{ current_source_symlink }}/application/config/{{ web_app_env }}/config.php"
+    dest: "{{ current_source_symlink }}/application/config/config.php"
     regexp: '(sess_use_database[^=]+)= TRUE;'
     replace: '\1= FALSE;'
 
 - name: Run migrations
-  command: php index.php migrate
+  command: php public/index.php migrate
```

Comment:
Based on the comment in the `.env` template, shouldn't we set `DB_DEBUG=TRUE` so that the migration fails on error, and shows that error? We don't want the app to continue if the schema is in an inconsistent state.

Reply:
I believe the error will still hit the logs, so I don't think we _need_ to show it.  But yeah, we need this command to return a failure somehow. I'll try to add a CLI exit status when it fails.

Reply:
I looked into this more and [the existing call to `show_error()`](https://github.com/GSA/project-open-data-dashboard/blob/2e376fbf3f3e3494da774335e3be0114fe02ee52/application/controllers/Migrate.php#L29) (as well as [the implicit call made due to `migration_auto_latest`](https://github.com/bcit-ci/CodeIgniter/blob/0db8b60492cc98ef3c85b986fc11e9dac6d7ca6c/system/libraries/Migration.php#L181)) already [results in a call to `exit()` with an error status](https://github.com/bcit-ci/CodeIgniter/blob/0db8b60492cc98ef3c85b986fc11e9dac6d7ca6c/system/core/Common.php#L404-L417), so processing stops at that point.